packages/graphql-yoga/src/index.ts

```diff
@@ -147,7 +153,7 @@ export class GraphQLServer extends BaseNodeGraphQLServer {
     this._server.close().then(
       () => {
         this.logger.info('Shutting down GraphQL server.')
-        process.exit(0)
+        // process.exit(0)
```

Comment:
not needed?

Reply:
yeah it should be removed. when I was testing this was causing more issues. I think you suggested in past but I missed it 😅

Reply:
```suggestion
```